packages/react-native/Libraries/AppDelegate/RCTLegacyInteropComponents.mm

```diff
@@ -11,7 +11,10 @@ @implementation RCTLegacyInteropComponents
 
 + (NSArray<NSString *> *)legacyInteropComponents
 {
-  return @[ @"RNTMyLegacyNativeView", @"RNTMyNativeView" ];
+  return @[
```

Comment:
This diff is not necessary. It is generated automatically by codegen (but it's committed to core with different formatting 😅)

Reply:
Ah I managed to get this off origin for so many commits and now it slipped :D